fitting/fitScanData.py

```diff
@@ -302,11 +302,11 @@ def fit(self, debug=False):
                     if self.isVFAT3:
                         fitTF1.SetParLimits(0, self.calDAC2Q_m[vfat]*(256)+self.calDAC2Q_b[vfat], self.calDAC2Q_m[vfat]*(1)+self.calDAC2Q_b[vfat])
                         fitTF1.SetParLimits(1, 0.0, self.calDAC2Q_m[vfat]*(128)+self.calDAC2Q_b[vfat])
-                        fitTF1.SetParLimits(2, -0.01, self.Nev[vfat][ch])
+                        fitTF1.SetParLimits(2, -0.01, self.calDAC2Q_m[vfat]*(1)+self.calDAC2Q_b[vfat])
                     else:
                         fitTF1.SetParLimits(0, -0.01, self.calDAC2Q_m[vfat]*(256)+self.calDAC2Q_b[vfat])
                         fitTF1.SetParLimits(1, 0.0,  self.calDAC2Q_m[vfat]*(128)+self.calDAC2Q_b[vfat])
-                        fitTF1.SetParLimits(2, -0.01, self.Nev[vfat][ch])
+                        fitTF1.SetParLimits(2, -0.01, self.calDAC2Q_m[vfat]*(256)+self.calDAC2Q_b[vfat])
```

Comment:
And did you see any real improvement to the fitting procedure with this change?

Reply:
No, I did not expected any change to the fitting procedure, but we just got lucky in some sense because the number of events we typically use (100) is larger than the pedestal values.